pymaker/governance.py

```diff
@@ -0,0 +1,136 @@
+# This file is part of Maker Keeper Framework.
+#
+# Copyright (C) 2019 EdNoepel
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import datetime
+from web3 import Web3
+
+from pymaker import Contract, Address, Transact
+from pymaker.auth import DSAuth
+
+
+# TODO: Complete implementation and unit test
```

Comment:
IMO we should add in Readme that the governance part is not really in scope of pymaker for now but any contribution to that welcomed